metacat-common/src/main/java/com/netflix/metacat/common/dto/notifications/sns/payloads/TablePartitionsUpdatePayload.java

```diff
@@ -35,19 +37,30 @@
 public class TablePartitionsUpdatePayload {
     private final int numCreatedPartitions;
     private final int numDeletedPartitions;
+    private final List<String> createdPartitionIds;
+    private final List<String> deletedPartitionIds;
+    private final String message;
```

Comment:
Make this a boolean field representing if the keys have been added or not.

Reply:
I want to give the consumer more information, if it's enabled also attached the number of partitionId. If exceeded, we let them know the threshhold.